public/stylesheets/cards.css

```diff
@@ -1,20 +1,71 @@
-.image-box {
-  width: 50vw;
+.card {
+  display: flex;
+  justify-content: center;
+  padding: 80px 10px;
+}
+.card-image {
+  background-color: black;
   max-width: 800px;
-  min-width: 320px;
-  padding: 40px 10px;
+  max-width: 100%;
+  width: 70vw;
+  min-width: 300px;
+  max-height: 450px;
+  min-height: 260px;
+  border-top-right-radius: 10px;
+  border-top-left-radius: 10px;
+  border-bottom-style: solid;
+  border: 1px black;
+}
+.card-box {
+  max-width: 800px;
+  width: 70vw;
+  min-width: 300px;
```

Comment:
I think it would be better to define min and max values for your card in a `div` container, you know, the most top level level DOM element that wraps your card so you don't have to define min max values for each element because they will scale to their parent.

Reply:
Noticed, will try to change that.